pkgs/top-level/all-packages.nix

```diff
@@ -17565,7 +17565,10 @@ with pkgs;
   gns3-gui = gns3Packages.guiStable;
   gns3-server = gns3Packages.serverStable;
 
-  gobject-introspection = callPackage ../development/libraries/gobject-introspection {
+  gobject-introspection = if (stdenv.hostPlatform != stdenv.targetPlatform)
```

Comment:
Is this a good place for something like this, as opposed to e.g. https://github.com/NixOS/nixpkgs/blob/a785565fb9f75c7361232b3da1c4384e8f0da8a2/pkgs/stdenv/cross/default.nix?